google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/AddHotelCalloutExtension.java

```diff
@@ -0,0 +1,268 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.extensions;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v2.common.HotelCalloutFeedItem;
+import com.google.ads.googleads.v2.enums.ExtensionTypeEnum.ExtensionType;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.resources.AdGroupExtensionSetting;
+import com.google.ads.googleads.v2.resources.CampaignExtensionSetting;
+import com.google.ads.googleads.v2.resources.CustomerExtensionSetting;
+import com.google.ads.googleads.v2.resources.ExtensionFeedItem;
+import com.google.ads.googleads.v2.services.AdGroupExtensionSettingOperation;
+import com.google.ads.googleads.v2.services.AdGroupExtensionSettingServiceClient;
+import com.google.ads.googleads.v2.services.CampaignExtensionSettingOperation;
+import com.google.ads.googleads.v2.services.CampaignExtensionSettingServiceClient;
+import com.google.ads.googleads.v2.services.CustomerExtensionSettingOperation;
+import com.google.ads.googleads.v2.services.CustomerExtensionSettingServiceClient;
+import com.google.ads.googleads.v2.services.ExtensionFeedItemOperation;
+import com.google.ads.googleads.v2.services.ExtensionFeedItemServiceClient;
+import com.google.ads.googleads.v2.services.MutateAdGroupExtensionSettingsResponse;
+import com.google.ads.googleads.v2.services.MutateCampaignExtensionSettingsResponse;
+import com.google.ads.googleads.v2.services.MutateCustomerExtensionSettingsResponse;
+import com.google.ads.googleads.v2.services.MutateExtensionFeedItemsResponse;
+import com.google.ads.googleads.v2.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * Adds a hotel callout to a specific account, campaign within the account, and adgroup within the
+ * campaign.
+ */
+public class AddHotelCalloutExtension {
+
+  private static class AddHotelCalloutParams extends CodeSampleParams {
+    @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+    private Long customerId;
+
+    @Parameter(names = ArgumentNames.CAMPAIGN_ID, required = true)
+    private Long campaignId;
+
+    @Parameter(names = ArgumentNames.AD_GROUP_ID, required = true)
+    private Long adGroupId;
+
+    @Parameter(names = ArgumentNames.CALLOUT_TEXT, required = true)
+    private String calloutText;
+
+    // IETF BCP 47 compliant language code
```

Comment:
As discussed, please find the place of language code list where we can refer to and put it here.